lmdeploy/serve/turbomind/triton_models/postprocessing/config.pbtxt

```diff
@@ -23,7 +23,7 @@ output [
 
 instance_group [
     {
-        count: 1
+        count: 16
```

Comment:
This might cause a core dump for some users.

Reply:
Which issue?